### PR TITLE
Register arty.is-a.dev

### PR DESCRIPTION
**Pull request by arty-airbourne**
Register arty.is-a.dev with CNAME record pointing to desariah-airbourne.github.io.

### CODE REVIEW

File: domains/arty.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "desariah-airbourne",
+           "email": "anthony@folley.io",
+           "discord": "1006588164003086497"
+        },
+    
+        "record": {
+            "CNAME": "desariah-airbourne.github.io"
```
Comment (CuteDog5695) [inline domains/arty.json:9]: 404 error page, please give a preview of your website

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/arty.json (hunk 1 above)
Comment (desariah-airbourne) [reply domains/arty.json:11]: Added GitHub repo 👍

**Review by arty-airbourne — COMMENTED** [review]
added repo url + description.

**Review by iostpa — CHANGES_REQUESTED** [review]
Site seems to not be accessible due to the CNAME file being added, checked the code in the repo and it just seems to say "is-a.dev". I'd like you to finish your website and remove the CNAME file until PR has been merged.

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**